fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/operations/CreateRdfSourceOperation.java

```diff
@@ -63,4 +71,9 @@ public String getParentId() {
     public void setParentId(final String parentId) {
         this.parentId = parentId;
     }
+
+    @Override
+    public ResourceOperationType getType() {
+        return CREATE;
```

Comment:
Same question about overriding the default implementation?

Reply:
valid point.